### DISCUSSION TITLE
An unknown error

### DISCUSSION THREAD

**Discussion by phungdaihiep79** [Web3Auth/Web3Auth#806] [Help]
Hello guys, I was deploying your product and encountered an unknown error and greatly annoyed our users.
![image](https://user-images.githubusercontent.com/60918515/199149612-6ebec8a2-8755-4c05-bbe9-26d4c4840807.png)

Here is my code
```
const web3Auth = new Web3Auth(WebBrowser, {
        sdkUrl: 'https://sdk.openlogin.com',
        clientId: TORUS_CLIENTID,
        network:
          Config.ENV === 'PRODUCTION'
            ? OPENLOGIN_NETWORK.MAINNET
            : OPENLOGIN_NETWORK.TESTNET,
        loginConfig: {
          [typeOfLogin]: {
            verifier: verifier,
            typeOfLogin: TYPEOFLOGIN.JWT,
            name: name,
            clientId: clientId,
            jwtParameters: {
              domain: domain,
              client_id: clientId,
              redirect_uri: redirect_uri,
              connection: connection,
            },
          },
        },
      })

      const state: State = await web3Auth.login({
        loginProvider: typeOfLogin,
        redirectUrl: REDIRECT_URL,
        mfaLevel: 'none',
      })
```

**Question:** Why does this error occur? and How to fix it?

**Comment by BboyStatix** [comment]
Do you have any logs?

**Reply by shahbaz17** [reply]
Also, please check out the code [here](https://web3auth.io/docs/sdk/react-native/custom-authentication#extraloginoptions-for-special-login-methods) to log in using JWT.

```js
const web3auth = new Web3Auth(WebBrowser, {
  clientId: "YOUR_CLIENT_ID", // web3auth's client id
  network: Network.TESTNET,
  loginConfig: {
    jwt: {
      verifier: "verifier-name", // get it from web3auth dashboard
      typeOfLogin: TypeOfLogin.JWT,
      name: "Custom Login with JWT",
      clientId: "YOUR_CLIENT_ID", // web3auth's client id
    },
  },
});
web3auth.login({
  loginProvider: LoginProvider.JWT,
  redirectUrl: resolvedRedirectUrl,
  extraLoginOptions: {
    domain: "example.com", // domain of your app backend
    id_token: "your JWT token",
  },
});
```

**Reply by phungdaihiep79** [reply]
i'm using auth0 for this code, not realy jwt

**Reply by shahbaz17** [reply]
To use Auth0, `loginProvider` needs to be JWT. And above code snippet should be used.

Check https://web3auth.io/docs/guides/auth0#initialising-the-openlogin-adapter

**Reply by phungdaihiep79** [reply]
I have implemented everything and everything works fine, but the situation like the picture above sometimes appears with greater frequency, the user has to cancel then log in again, everything works again.

**Reply by phungdaihiep79** [reply]
this creates confusion for some users who fall into this case

**Reply by YZhenY** [reply]
we need logs to understand what is the error you are facing

**Reply by phungdaihiep79** [reply]
@YZhenY  This case has appeared on some users and we have got feedback, we have tried to reproduce it but it still does not work, the number of users experiencing this case is increasing and we find it very difficult bear with this situation.

**Comment by phungdaihiep79** [comment]
Could you explain why we have this situation?
